### PR TITLE
Update Controllers examples to apply changed UX

### PR DESCRIPTION
**Pull request by singa2000**
[Issue] N/A
[Problme] Controllers examples are weird.
[Solution] Update contollers examples to UX List.
           And move unused samples in the controllers to 'content' examples for reference.

Signed-off-by: cookie <cookie@samsung.com>

### CODE REVIEW

File: examples/mobile/UIComponents/components/content/index.html [hunk 1]
```diff
@@ -23,6 +23,46 @@ <h1>
...
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="searchbar/index.html">
+						Search Bar
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="progress/index.html">
+						Progress &amp; Activity
```
Comment (TomaszLukawskiSam) [inline examples/mobile/UIComponents/components/content/index.html]: Progress has own category in new Guideline.

File: examples/mobile/UIComponents/components/content/index.html [hunk 2]
```diff
@@ -23,6 +23,46 @@ <h1>
...
 					</a>
 				</li>
+				<li class="ui-li-anchor">
+					<a href="floatingactions/index.html">
+						Floating Actions
+					</a>
+				</li>
+				<li class="ui-li-anchor">
+					<a href="textinput.html">
+						Text Field
```
Comment (TomaszLukawskiSam) [inline examples/mobile/UIComponents/components/content/index.html:33]: Text field has own category in new guideline.

File: examples/mobile/UIComponents/components/content/index.html (hunk 1 above)
Comment (singa2000) [reply examples/mobile/UIComponents/components/content/index.html]: Yes. I know.
Now some Progress examples are not working well, so now I don't make their category.
I'll update the category in the next PR.
Thanks for your comments.

File: examples/mobile/UIComponents/components/content/index.html (hunk 2 above)
Comment (singa2000) [reply examples/mobile/UIComponents/components/content/index.html:33]: Yes. I know.
Now some Text field examples are not working well, so now I don't make their category.
I'll update the category in the next PR.
Thanks for your comments.

File: examples/mobile/UIComponents/components/content/index.html (hunk 1 above)
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/content/index.html]: not need to &amp;
I think that you input only Progress.

File: examples/mobile/UIComponents/components/controllers/index.html [hunk 3]
```diff
@@ -28,69 +28,24 @@ <h1>
...
-						Search Bar
+					<a href="master-onoffswitch.html">
+						Master On/Off switch
 					</a>
 				</li>
 				<li class="ui-li-anchor">
-					<a href="progress/index.html">
-						Progress &amp; Activity
+					<a href="onoffswitch.html">
+						On/Off switch
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/controllers/index.html:43]: On/Off switch is correct?
it is already input Progress in examples/mobile/UIComponents/components/content/index.html file.
are you will remove this file?

**Review by DohyungLim — COMMENTED** [review]
2 comment.

File: examples/mobile/UIComponents/components/content/index.html (hunk 1 above)
Comment (singa2000) [reply examples/mobile/UIComponents/components/content/index.html]: Done

**Review by DohyungLim — APPROVED** [review]
LGTM.

File: examples/mobile/UIComponents/components/controllers/index.html (hunk 3 above)
Comment (singa2000) [reply examples/mobile/UIComponents/components/controllers/index.html:43]: On/Off switch is just located in the examples/mobile/UIComponents/components/controllers/index.html file.
There isn't in the content example.

**Review by lslachciak — APPROVED** [review]
verified,works well
but we will need more PR to make more order in main menu of UIComponents

**Comment by singa2000** [comment]
Sure.
I'll update more in the next PR.

**Pull request merged**